<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>329</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>329</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="mOptionsLayout">
   <item row="2" column="0">
    <widget class="QFrame" name="mButtonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="mButtonFrameLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="mHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mAcceptButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mRejectButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mApplyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="mTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTab01">
      <attribute name="title">
       <string>Auto-Completer</string>
      </attribute>
      <layout class="QGridLayout" name="mTab01Layout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="mSourceGroup">
         <property name="title">
          <string>Source</string>
         </property>
         <layout class="QGridLayout" name="mSourceGroupLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="mSourceLineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="mSourceButton1">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="mSourceButton2">
            <property name="text">
             <string>parse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="mSourceButton3">
            <property name="text">
             <string>clean</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="mOptionsGroup">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="mOptionsGroupLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="mMaxVisibleLabel">
            <property name="text">
             <string>Max visible items</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="mMaxVisibleBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mMinCharLabel">
            <property name="text">
             <string>Min character number</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="mMinCharBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mTypingLabel">
            <property name="text">
             <string>Typing sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="mSortingLabel">
            <property name="text">
             <string>Sorting sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="mTypingComboBox">
            <item>
             <property name="text">
              <string>case-sensitive</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>case-insensitive</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="mSortingComboBox">
            <item>
             <property name="text">
              <string>case-sensitive</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>case-insensitive</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>unsorted</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTab03">
      <attribute name="title">
       <string>Field</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <spacer name="mVerticalSpacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="mCodeGroup">
         <property name="title">
          <string>Code</string>
         </property>
         <layout class="QGridLayout" name="mCodeGroupLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="mTabSizeLabel">
            <property name="text">
             <string>Tab Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="mTabSizeBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mGuideLineLabel">
            <property name="text">
             <string>Guide Line</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="mGuideLineBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTab02">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QGridLayout" name="mTab02Layout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="mApplicationGroup">
         <property name="title">
          <string>Application</string>
         </property>
         <layout class="QGridLayout" name="mApplicationLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="mInstanceLabel">
            <property name="text">
             <string>Instance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mInstanceComboBox">
            <item>
             <property name="text">
              <string>single</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>multiple</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mLanguageLabel">
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="mLanguageComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="mVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
